src/test/java/com/ratifire/devrate/service/interview/InterviewFeedbackDetailServiceTest.java

```diff
@@ -0,0 +1,125 @@
+package com.ratifire.devrate.service.interview;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.ratifire.devrate.dto.InterviewFeedbackDetailDto;
+import com.ratifire.devrate.entity.Mastery;
+import com.ratifire.devrate.entity.Skill;
+import com.ratifire.devrate.entity.interview.Interview;
+import com.ratifire.devrate.entity.interview.InterviewFeedbackDetail;
+import com.ratifire.devrate.entity.interview.InterviewRequest;
+import com.ratifire.devrate.exception.InterviewFeedbackDetailNotFoundException;
+import com.ratifire.devrate.mapper.DataMapper;
+import com.ratifire.devrate.repository.interview.InterviewFeedbackDetailRepository;
+import com.ratifire.devrate.service.specialization.SkillService;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class InterviewFeedbackDetailServiceTest {
+
+  @Mock
+  private InterviewFeedbackDetailRepository feedbackDetailRepository;
+
+  @Mock
+  private DataMapper<InterviewFeedbackDetailDto, InterviewFeedbackDetail> feedbackDetailMapper;
+
+  @Mock
+  private SkillService skillService;
+
+  @InjectMocks
+  private InterviewFeedbackDetailService feedbackDetailService;
+
+  private InterviewFeedbackDetail feedbackDetail;
+  private InterviewFeedbackDetailDto feedbackDetailDto;
+
+  private static final long FEEDBACK_ID = 1L;
+  private static final List<Long> SKILL_IDS = List.of(101L, 102L);
+
+  @BeforeEach
+  void setUp() {
+    feedbackDetail = InterviewFeedbackDetail.builder()
+        .id(FEEDBACK_ID)
+        .startTime(ZonedDateTime.now())
+        .skillsIds(SKILL_IDS)
+        .build();
+
+    feedbackDetailDto = InterviewFeedbackDetailDto.builder()
+        .interviewStartTime(feedbackDetail.getStartTime())
+        .skills(List.of())
+        .build();
+  }
```

Comment:
Can you please remove the `setUp` method? The variables `feedbackDetail` and `feedbackDetailDto` are only used in one test method. Currently, the variables from the `setUp` method are created for every test case, which seems redundant.
Just remove `setup` method and move variables from it to the `getInterviewFeedbackDetailWhenExistsThenReturnFeedbackDetailDto` test case